client/client.go

```diff
@@ -364,6 +366,23 @@ func (c *Client) ListUsers() (*http.Response, error) {
 	return c.get("/user/list", nil)
 }
 
+// EnableTotp enables Totp for a given user
+func (c *Client) EnableTotp(username, password string) (*http.Response, error) {
+	return c.post("/user/totp/enable", &common.UserRequest{
+		Username: username,
+		Password: password,
+	})
+}
+
+// DisableTotp enables Totp for a given user
+func (c *Client) DisableTotp(username, password, totp string) (*http.Response, error) {
```

Comment:
This is a bug. `DisableTotp` should take no arguments because the user is authenticated with his/her access token when making the request.

Reply:
I'm fixing it now.